internal/hclutil/hclutil.go

```diff
@@ -0,0 +1,63 @@
+// Copyright 2020 Google LLC
```

Comment:
Please add tests, seems like it shouldn't be too difficult.

Reply:
Done